README.md

```diff
@@ -32,6 +32,26 @@ vagrant up
 ```
 [localhost:8000](http://localhost:8000)
 
+Develop locally using Procboy
+=============================
+
+Procboy runs multiple processes, configured in Procfile, using environment variables configured in .env:
+
+Prepare dependencies and configure variables (preferably in a virtualenv[1]):
+```
+pip install -r requirements.txt
+npm install
+cp .env.template .env
+```
+Configure REMOTE_USER
```

Comment:
Could say `Configure REMOTE_USER in .env` here, otherwise LGTM (feel free to push to master).